api/open-positions.ts

```diff
@@ -0,0 +1,47 @@
+import Airtable from "airtable";
+import { NowRequest, NowResponse } from "@now/node";
+
+interface JobRecord {
+  "Stav": "Aktivní" | "Obsazeno";
+  "Úkol/Role": string;
+  "Odkaz na celou příležitost": string | null;
+  "Detaily": string | null;
+  "Časová náročnost": string | null;
+  "Urgentní poptávka": boolean;
+}
+
+interface Position {
+  title: string;
+  isOpen: boolean;
+  isUrgent: boolean;
+  detailUrl: string | null;
```

Comment:
obecně preferuji `detailUrl?: string` místo `|null`, ale YMMV :-)

Reply:
_Vyřešeno ve mém PR._

Reply:
Ono to není totéž, je? Opět se dostáváme k tomu, že tomu jazyku nerozumím :), ale když to napíšu takhle (tedy `detailUrl?: string`, což bych taky preferoval), tak mě pak peskuje u tohoto:

```
function parseRecord(record: Airtable.Record<JobRecord>): Position {
  const f = record.fields;
  return {
    title: f["Úkol/Role"],
    isOpen: f["Stav"] === "Aktivní",
    isUrgent: f["Urgentní poptávka"],
    detailUrl: f["Odkaz na celou příležitost"], // Type 'string | null' is not assignable to type 'string | undefined'.
    description: f["Detaily"],
    timeRequirements: f["Časová náročnost"],
  };
}
```

Takže by tam muselo být `f["Odkaz na celou příležitost"] || undefined`? 🤷

Reply:
🤦 Sorry, přehlídl jsem, jaké to má konsekvence. Muselo by to být tak, jak píšeš. Jestli to je lepší nebo horší než `| null`... říkám rozhodné, rázné, chlapské NEVÍM :-D

Reply:
Tak pojďme zpátky k `| null`, pokrok nezastavíš 😈